sleap/nn/evals.py

```diff
@@ -724,3 +724,22 @@ def evaluate_model(
         logger.info("OKS mAP: %f", metrics["oks_voc.mAP"])
 
     return labels_pr, metrics
+
+
+def load_metrics(model_path: str, split: str = "val") -> Dict[str, Any]:
+    """Load metrics for a model.
+
+    Args:
+        model_path: Path to a model folder.
+        split: Name of the split to load the metrics for. Must be `"train"`, `"val"` or
+            `"test"` (default: `"val"`).
+
+    Returns:
+        The loaded metrics as a dictionary.
+    """
+    if os.path.isdir(model_path):
+        metrics_path = os.path.join(model_path, f"metrics.{split}.npz")
+    else:
```

Comment:
Suggestion: either make 'split' optional (and require a path to file in this case), or require that model_path is a folder. It is a bit loose when e.g. a 'test' metrics file can be loaded with 'split=val'

Reply:
Yeah good point, added some info in the docstring